### ISSUE TITLE
STDIN in -draw 'image over...' returns improper image header error

### ISSUE THREAD

**Issue by harvey-sg** [ImageMagick/ImageMagick#3700]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported
- [X] I have verified that I am using the latest version of ImageMagick

### ImageMagick version

7.0.11-13

### Operating system

MacOS

### Operating system, version and so on

10.14.6

### Description

Using STDIN in `-draw 'image over 0,0 100,100 "-"'` returns the errors
```
magick: improper image header `/var/folders/gx/5jh3kbx95f311w5g_9spr4pr0000gn/T/magick-igKP2U-wVdCr2gxN5y9zvpCtKhRVuO8R' @ error/png.c/ReadPNGImage/4286.
magick: non-conforming drawing primitive definition `image' @ error/draw.c/RenderMVGContent/4470.
```
This worked for me with version 7.0.10-34, it doesn't on 7.0.11-13.
Anybody else having the same issue?

### Steps to Reproduce

`magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 0,0 100,100 "-"' zzzz.png`

**Comment by fmw42** [comment]
Standard in should be `@-` 

So try

`magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 0,0 100,100 "@-"' zzzz.png`

**Comment by harvey-sg** [comment]
Hi @fmw42 , with the `@-`:
```
magick: unable to open image '@-': No such file or directory @ error/blob.c/OpenBlob/3537.
magick: non-conforming drawing primitive definition `image' @ error/draw.c/RenderMVGContent/4470.
```
What to be updated in the policy.xml ?

**Comment by fmw42** [comment]
Yes. Edit your policy.xml file and uncomment the line for the @ and give it "read|write" permission.

**Comment by harvey-sg** [comment]
policy.xml is now:
```
<policymap>
  <policy domain="path" rights="read|write" pattern="@*" />
</policymap>
```

Still getting the error:
```
magick: unable to open image '@-': No such file or directory @ error/blob.c/OpenBlob/3537.
magick: non-conforming drawing primitive definition `image' @ error/draw.c/RenderMVGContent/4470.
```
What am I missing?

**Comment by harvey-sg** [comment]
Also without the suggested policy change, the below use of STDIN does work with version `7.0.11-13`
```
magick rose: gif:- | magick - -resize "200%" bigrose.jpg
```

**Comment by fmw42** [comment]
That is a different case. But I tested your original command and get the same error. I think IM is not set up to allow @- as the reference to the composite image in -draw. One of the IM developers will need to look into this.
